### ISSUE TITLE
Make sure the scanning errors are making their way to the client

### ISSUE THREAD

**Issue by Krakaw** [big-neon/bn-api#624]
Specifically in this issue revelrylabs/bn-mobile-react#201 the returned error message for an already scanned ticket is `We found an Issue` instead of `Already Redeemed`. 

The same thing happened [here](https://github.com/revelrylabs/bn-mobile-react/issues/238#issuecomment-443224446)

**Comment by Timo614** [comment]
@Krakaw could that be something from bn-api-node? 

I just tried locally after setting up a similar situation and it's showing me an error message similar to what you mentioned it should show:

```
curl --request POST --data '{"redeem_key": "WUFY92V6N"}' http://localhost:9000/events/ed26e974-c2ab-4fc2-bb38-527a77a0b8a1/redeem/16811cf0-5d1a-4838-a6cf-2993ac00683e --header "Content-Type: application/json" --header "Authorization: Bearer eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJzdWIiOiJmMWNhZTFiYi01ZmM5LTRmYWUtYTY0OC02OTU5MzRhNjQ1YjkiLCJpc3MiOiJiZy1zYW1wbGUtaXNzdWVyIiwiZXhwIjoxNTQzODYyOTg3fQ.xQDYf41RgYVcLcZ-VYb35L62zGwr2AMhYelPAirNOi0" -i
HTTP/1.1 409 Conflict
content-length: 45
content-type: application/json
access-control-allow-origin: http://localhost:3000
vary: Origin
x-app-version: 0.1.0
date: Mon, 03 Dec 2018 18:36:48 GMT

{"error":"Ticket has already been redeemed."}
```

**Comment by mikethetike** [comment]
I made some unrelated fixes to this code when I had a failing test for #623. Might be worth checking if the code I put in fixes this.

**Comment by Timo614** [comment]
I tried to see if I could find anything in the code for bn-api-node or bn-api that could cause this sort of issue but I think it's like Mike said in the IRC room perhaps it was related to the issue he fixed in #623 and one of the apps along the way returning a nicer error message by default.

I'd think though if they still see it it's probably something happening outside of bn-api as I don't see that particular error message anywhere in our code. Was thinking perhaps something was behaving badly with 409 errors and ignoring the error json we return but Mike brought up the point that the other match arm through a 500 error so perhaps it was something like that.

Let me know though and can take another look if they think it is coming from bn-api just not sure how it'd happen from taking a glance at it's code and trying it out a bit.

**Comment by Krakaw** [comment]
Issue is resolved
```
curl 'http://localhost:9000/events/123e8910-bffe-4e5c-a69b-e70e093fd8e6/redeem/2c939d0b-ed2c-4d0d-81b0-16f7a122891c' -H 'Pragma: no-cache' -H 'Origin: http://localhost:3000' -H 'Accept-Encoding: gzip, deflate, br' -H 'Accept-Language: en-GB,en-US;q=0.9,en;q=0.8' -H 'Authorization: Bearer eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJzdWIiOiIwYTY1YjhlNC1iZmQyLTQwNjMtOWM5Yi0zNDAzOGUyYzY2ODkiLCJpc3MiOiJ0ZW1wIiwiZXhwIjoxNTQ2Njk3NDk3fQ.o7liIVLxcQxkAeWsb9-IF1QEa7jOapGAW7xPScakPDE' -H 'Content-Type: application/json' -H 'Accept: application/json, text/plain, */*' -H 'Cache-Control: no-cache' -H 'User-Agent: Mozilla/5.0 (Macintosh; Intel Mac OS X 10_14_2) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/71.0.3578.98 Safari/537.36' -H 'Connection: keep-alive' -H 'Referer: http://localhost:3000/box-office/guests' -H 'X-API-Client-Version: bn-api-node@0.3.25' --data-binary '{"redeem_key":"FCXKSADZR"}' --compressed
{"error":"Ticket has already been redeemed."}
```
